buildcmake

```diff
@@ -131,7 +131,10 @@ opt_shmem=0
 opt_xpmem=0
 opt_qlogic=0
 opt_randomized_msgq=0
-opt_reconverse=0
+opt_reconverse=1 # fake argument for now
```

Comment:
Does it make sense to add this as an actual option (like `--reconverse` or so)

Reply:
I think we already have the `--reconverse` option. Since this branch only works for the reconverse build now, I just set it to 1 by default to simplify things. In the long run, I would recommend adding a new NETWORK type called `reconverse` and making it an alias of `multicore` plus enabling this option.